components/03-modules/form/_form.scss

```diff
@@ -1,11 +1,12 @@
 .form {
     &--compact {
-        @include mq($screen-s) {
+        @include mq($screen-m) {
             display: flex;
             justify-content: center;
         }
 
         .form__heading {
+            display: none;
```

Comment:
By default it's hidden because we don't need it for regular compact form . It'll be visible for footer. You said I can add !important for override it in footer.